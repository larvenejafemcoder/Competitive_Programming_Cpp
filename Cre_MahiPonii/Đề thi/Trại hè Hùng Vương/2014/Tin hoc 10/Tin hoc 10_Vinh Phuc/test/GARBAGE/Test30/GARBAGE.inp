WDUUWUDEESDUEE
WSUUSUUUEDSSDDEDE
WWNUSWDEESUN
SEDSN

WUWNDWUEN
N 17
