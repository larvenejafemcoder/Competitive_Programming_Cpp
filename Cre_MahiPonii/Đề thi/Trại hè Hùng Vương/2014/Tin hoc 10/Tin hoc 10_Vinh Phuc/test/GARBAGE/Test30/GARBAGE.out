1321091217465608
